sdk/iothub/azure-iot-deviceprovisioningservice/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (2023-04-30)
+
```

Comment:
Reminder to add at least you and me as codeowners once root sdk path is confirmed.